### ISSUE TITLE
Variant of scikit-image.feature.hog (Felzenszwalb et al. paper)

### ISSUE THREAD

**Issue by cissealine** [scikit-image/scikit-image#7016]
### Description:

Hi,

I am really interesting in using scikit-image.feature.hog for data analysis (thanks again for this wonderful documentation by the way !). I saw that you implemented this algorithm based on the Dalal, Triggs (2005) article.

I would like to know if you plan to implement another version of scikit-image.feature.hog, but based on Felzenszwalb et al. 2010 article (DOI : doi 10.1109/TPAMI.2009.167), similar to Matlab's vlfeat package ('UoCTTI' variant, which is the default compared to 'DalalTriggs' variant). 

On Matlab, better results were observed in some analyses with this variant, compared to the original (Dalal, Triggs) implementation, and I am interesting in reproducing that in Python. 
If that's not in your short-term plans, I'll have to think about implementing it from scratch, but it is still in discussion (due to time reasons). 

Thanks by advance for your answer !
Have a nice day,
All the best

**Comment by stefanv** [comment]
Hi @cissealine, thanks for this request. We don't have any immediate plans to work on this feature, but we would welcome it as a contributed addition.

**Comment by lagru** [comment]
Hey @cissealine, did you ever get around to implementing this? :)

**Comment by cissealine** [comment]
Hi everyone, 

Thanks to you for all your answers :D Unfortunately, I did not have time to work on it, and I do not think I could... 

However, this implementation was coded in C in the VLFeat package for Matlab (which is open-source), and I found a repo GitHub which wraps it using Cython (here : https://github.com/menpo/cyvlfeat). This repo still works but is quite old and not maintained. 

As the base code of VLFeat already implements a lot of features in C, probably the wrapping remains a good idea (correct me if I am wrong ^^). 

Thanks a lot, 
All the best
